inst/rmarkdown/templates/inbo_boek/skeleton/04-doing-more.Rmd

```diff
@@ -0,0 +1,160 @@
+
+Cross-referencing a specific position implies the need for a label at that position, which can be linked to from another place in the text. At the moment, an output format specific way of inserting labels is needed for both HTML and PDF based output formats (including epub format). Both ways can be combined in your Rmd-files however, because the pandoc converter -- which interprets the *.md file and converts it -- will recognize 'which is which':
+
+- label for PDF output formats: insert `\hypertarget{your_unique_label}{}`
+- label (anchor) for HTML output formats: insert `<A id="your_unique_label"></A>`
+- combination: insert `\hypertarget{your_unique_label}{}<A id="your_unique_label"></A>`
+
+Referencing the label can be done in an output format agnostic way, using the markdown approach for hyperlinks: `[linkable words that are visible](#your_unique_label)`. For example, click [here](#example) to jump to the bullet on how to cross-reference a part.
+
+
+
+## Inserting quotes
+
+You can insert a quote block using `>` before each line:
+
+> Linus Torvalds said:
+>
+> In real open source, you have the right to control your own destiny.
+>
+> _(Source: <https://www.brainyquote.com/quotes/quotes/l/linustorva587384.html>)_
+
+
+
+## Grouping chapters as parts
+
+Sometimes you have many chapters and you want to group them into parts. Bookdown offers a unified approach for PDF and HTML output formats:
+
+- just insert a chapter, with a prefix `(PART)` and a suffix `{-}` -- indicating that it is a part and that it must not be numbered as a chapter, respectively. For example:
+
+    ```
+    # (PART) The title of my first part {-}
+    
+    # The title of the first chapter in the part
+    
+    Text, sections, subsections etc.
+    ```
+    
+    A label for the part will be automatically generated^[
+    The automatic labelling actually occurs for all headers, and is accomplished by pandoc markdown's [autoidentifier](http://pandoc.org/MANUAL.html#header-identifiers) feature!],
+    during processing of the file. For the example above, it will be: `part-the-title-of-my-first-part`.
+    
+- \hypertarget{example}{}<A id="example"></A>you can therefore cross-reference the part in the ordinary bookdown way: `see part \@ref(part-the-title-of-my-first-part)`.
+
+
+
+
+## Adding appendices
+
+The user can handle appendices identically to chapters; see section \@ref(S:crossref). Because they always appear after the regular chapters, and no regular chapters occur after them, it suffices to insert an unnumbered `(APPENDIX)` "chapter" _before the title of the first appendix_:
+
+    # (APPENDIX) Appendices {-}
+    
+    # Title of first appendix
+    
+    Contents of first appendix
+    
+    # Title of second appendix
+    
+    Contents of first appendix
+
+The consequence of handling appendices separately, is that they will be numbered as A, B, C and so on.
+
+
+
+## Corporate style blocks in both HTML and PDF {#S:htmlblocks}
```

Comment:
This section is a partial duplicate of a section in 03-corporate_identity.Rmd. I'd combine them.